lib/pubid/iso/amendment.rb

```diff
@@ -1,9 +1,34 @@
 module Pubid::Iso
   class Amendment < Supplement
-    def render_pubid
+    # @param stage_format [:short,:long] format for stage rendering
+    # @param with_date [Boolean] include date
+    def render_pubid(stage_format = :long, with_date = true)
```

Comment:
@mico can you help validate the `stage_format` so it must either be `:long` or `:short`? We don't want codes that are open ended.